### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by magicus**
Jesper asked me to remove those files :). If I do this before splitting index.md, that patch gets easier.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [ ] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/guide pull/83/head:pull/83` \
`$ git checkout pull/83`

Update a local copy of the PR: \
`$ git checkout pull/83` \
`$ git pull https://git.openjdk.java.net/guide pull/83/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 83`

View PR using the GUI difftool: \
`$ git pr show -t 83`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/guide/pull/83.diff">https://git.openjdk.java.net/guide/pull/83.diff</a>

</details>

### CODE REVIEW

File: src/codeReview.md [hunk 1]
```diff
@@ -1,117 +0,0 @@
-% Community Code Review
```
Comment (irisclark) [inline src/codeReview.md:1]: The contents of this file was never integrated into the Guide so I don't think it should be removed quite yet.

Comment (JesperIRL) [reply src/codeReview.md:1]: I have a local copy of that content that I'll be integrating shortly so removing the file now is fine. I did however notice that the URL cr.openjdk.java.net is redirected to this file, so before we push this change to the web server we need to change that to point into the guide.

**Comment by magicus** [comment]
Sounds like it's better to push the split as it is now, and remove these files later, after you've updated the cr link and pushed your local copy.

**Comment by magicus** [comment]
Withdrawing for now. Let's revisit this cleanup later.

**Pull request closed without merging**